activity_calendar/static/js/fullcalendar-init.js

```diff
@@ -86,17 +86,11 @@ function onEventClick(info, calendar) {
         }
     } else {
         // 5 augustus, 19:30 – 6 augustus, 02:00
-        var opts = {}
-
-        if (!event.allDay) {
-            opts = {hour: 'numeric', minute: 'numeric'}
-        }
-
         date_str = start_date.toLocaleString('en-gb', {
-            month: 'long', day: 'numeric', ...opts
+            month: 'long', day: 'numeric', hour: 'numeric', minute: 'numeric',
         })
         date_str += ' - ' + end_date.toLocaleString('en-gb', {
-            month: 'long', day: 'numeric', ...opts
```

Comment:
Maybe we should add an ES6 toolchain.

I can probably port that over from abot (I should maybe start porting over some stuff from abot anyway)